src/persist-client/src/write.rs

```diff
@@ -215,7 +215,7 @@ where
         self.machine.applier.clone_upper()
     }
 
-    /// Fetches and returns a recent shard-global `upper`. Importantly, this operation is not
+    /// Fetches and returns a recent shard-global `upper`. Importantly, this operation is
     /// linearized with other write operations.
```

Comment:
while you're in here: s/with other write/with write/

Reply:
Done